css/index.css

```diff
@@ -143,4 +145,215 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+
+/* Index Page Katie's Classes */
+#box1 {
```

Comment:
this is a perfectly fine approach, I am just mentioning to people that I want you to understand how you could also do this by using two classes and greater specificity rather than IDs. If you have questions on this, please DM me!!